src/auth/signup.js

```diff
@@ -0,0 +1,164 @@
+const InputMessage = {
+  ERROR_EMPTY_EMAIL: "이메일을 입력해주세요.",
+  ERROR_INVALID_EMAIL_FORMAT: "올바른 이메일 주소가 아닙니다.",
```

Comment:
개인취향으로 모두 에러 메세지라면 아래처럼 사용해도 좋을거 같습니다 ㅎㅎ

```suggestion
const INPUT_ERROR_MESSAGE = {
  EMPTY_EMAIL: "이메일을 입력해주세요.",
  INVALID_EMAIL_FORMAT: "올바른 이메일 주소가 아닙니다.",
```